### DISCUSSION TITLE
Webpack refreshes page on submit?

### DISCUSSION THREAD

**Discussion by Heilemann** [react-hook-form/react-hook-form#6940] [Q&A] [answered]
Hi. I just started using react-hook-form, and I'm having a problem where the page is refreshed when the form submits. As far as I can tell that shouldn't be happening. I tried tracing the issue and ended up somewhere in web pack, and [this SO question](https://stackoverflow.com/questions/64090974/page-gets-reloaded-on-form-submit-why) seems to verify that it's a hot reloading problem. Hopefully I don't have to turn off hot reload o use react-hook-form?

I'm not using TS. There's nothing funky about my setup that I can tell. 

`e.preventDefault()` in `<form onSubmit={e => { e.preventDefault(); handleSubmit(onSubmit)}}>` seems to fix this, but I'm not seeing that in the examples, and as far as I could tell the `handleSubmit` code has that built in.

But the `onSubmit` call never goes through?

**Update:** Nevermind, I found the issue (I was screwing up the call to handleSubmit!)

**Comment by bluebill1049** [comment, marked as answer]
cool! glad you found the solution.

**Reply by snaerth** [reply]
@Heilemann can you share your solution?. I am having the same issue

**Reply by Heilemann** [reply]
I... don't remember 😄. But look at how you're calling handleSubmit. Maybe it's 'handleSubmit()' or '() => handleSubmit()' or some such?

**Reply by fahadsohail482** [reply]
@snaerth Please read this [comment](https://github.com/react-hook-form/react-hook-form/discussions/6745#discussioncomment-1444947) for solution.

**Comment by matija2209** [comment]
I had the same problem. The issue was that I had another form wrapping my react hook form. Submit btn was triggering the parent form.

**Reply by leapful** [reply]
@matija2209 Please create another discussion for your issue. This current discussion is in a long time ago and might not be related to your problem.
